pkg/registry/service/ip_allocator.go

```diff
@@ -174,6 +174,15 @@ func (ipa *ipAllocator) AllocateNext() (net.IP, error) {
 	return nil, fmt.Errorf("can't find a free IP in %s", ipa.subnet)
 }
 
+func GenerateFirstTwoIPsFromSubnet(subnet *net.IPNet) (net.IP, net.IP, error) {
```

Comment:
1) Add a TODO to refactor the IP-math logic some place more generic.

2) How about GetIndexedIP(subnet *net.IPNet, index int) and then call it twice?